src/plugins/solvers/linear_solver.cpp

```diff
@@ -552,7 +553,7 @@ namespace phylanx { namespace execution_tree { namespace primitives
                             "the linear_solver_operation primitive "
                             "requires "
                             "that first operand to be a matrix and "
-                            "the second "
+                            "the secotond "
```

Comment:
This was most likely an accidental change.

Reply:
Yes. I will change it.